docs/snippets/common/storybook-coverage-addon-install.npm.js.mdx

```diff
@@ -0,0 +1,3 @@
+```shell
+npm install nyc @storybook/addon-coverage --save-dev
```

Comment:
nyc is a dependency of the test-runner so users will already have it in their projects

Reply:
Fixed with the latest commit.